tensorflow/lite/toco/graph_transformations/identify_prelu.cc

```diff
@@ -55,18 +55,45 @@ ::tensorflow::Status IdentifyPRelu::Run(Model* model, std::size_t op_index,
   }
 
   const auto* relu_input_op = GetOpWithOutput(*model, add_op->inputs[0]);
-  if (relu_input_op == nullptr || relu_input_op->type != OperatorType::kRelu ||
-      relu_input_op->inputs.size() != 1 ||
+  const auto* mul_op = GetOpWithOutput(*model, add_op->inputs[1]);
+  // this can be commutative so we need to find which operator is relu and which
+  // one is mul.
+  if (mul_op == nullptr || relu_input_op == nullptr) {
+    return ::tensorflow::Status::OK();
+  }
+
+  if (relu_input_op->type != OperatorType::kRelu &&
```

Comment:
I think the entire code from line 65 to line  76 can be simplified as:

    if (relu_input_op->type == OperatorType::kMul &&
        mul_op->type == OperatorType::kRelu) {
      std::swap(relu_input_op, mul_op);
    }
    if (relu_input_op->type != OperatorType::kRelu ||
      mul_op->type != OperatorType::kMul) {
      return ::tensorflow::Status::OK();
    }

Also note you can use `std::swap` instead of the 3 lines swap implementation.

Reply:
@miaout17 , thanks for the comments, i have updated the code as per your suggestions.

Regards
Amit